MSAL/test/integration/native_auth/requests/reset_password/MSALNativeAuthResetPasswordChallengeIntegrationTests.swift

```diff
@@ -39,7 +39,7 @@ final class MSALNativeAuthResetPasswordChallengeIntegrationTests: MSALNativeAuth
         )
 
         sut = try provider.challenge(
-            token: "<password-reset-token>",
+            token: "<passwordResetToken>",
```

Comment:
nit: wouldn't it fit better `"<continuation-token>"`? (same for the rest of the tests)